tests/intern.js

```diff
@@ -25,17 +19,28 @@ define(["intern"], function(intern, topic) {
         : 10000
     },
 
+    setup: function() {
+      define("FunctionalHelpers", ["tests/functional/lib/helpers"], function(
+        FunctionalHelpers
+      ) {
+        return {
+          checkServer: FunctionalHelpers.checkServer
+        };
+      });
+      var serverPromise;
+      require(["FunctionalHelpers"], function(FunctionalHelpers) {
+        serverPromise = FunctionalHelpers.checkServer();
+      });
+      return serverPromise;
+    },
+
     // The port on which the instrumenting proxy will listen
     proxyPort: 9090,
 
     // A fully qualified URL to the Intern proxy
     proxyUrl: "http://127.0.0.1:9090/",
     siteRoot: siteRoot,
     tunnel: "SeleniumTunnel",
-    tunnelOptions: {
```

Comment:
Can you explain this part to me? Why don't we want to specify the drivers?

Reply:
That was a mistake. I removed it while testing something, but should have put it back. Thanks for noticing.